docs/34-national-clouds.md

```diff
@@ -0,0 +1,30 @@
+# Configuration for national clouds
+
+By default, doAzureParallel is configured to run in public Azure cloud, if you want to run workload in national clouds, you can configure endpoint suffix for storage account in cluster config, it tells doAzureParallel which national cloud environment the storage account resides.
```

Comment:
Update to:

doAzureParallel is configured to run in public Azure cloud by default. To run workloads in national clouds, configure endpoint suffix for storage account in the cluster config which tells doAzureParallel which national cloud environment the storage account resides.

Reply:
updated.